src/priya_t/Assignment27_RemoveUppercaseFromString.java

```diff
@@ -0,0 +1,23 @@
+package priya_t;
+
+public class Assignment27_RemoveUppercaseFromString {
+
+	String str = "TechNoCREdits";
+
+	void removeuppercase() {
+
+		for (int i = 0; i < str.length(); i++) {
+            char ch = str.charAt(i);
+			boolean flag = Character.isUpperCase(ch);
+			if (flag == true) {
+			}
+		}
+		String finalStr = str.replaceAll("[TNCRE]", "");
```

Comment:
You cannot write hardcoded like this, make a method that will use for all strings